srcmv3/go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/confluentinc/ccloud-sdk-go-v2/srcmv3
```

Comment:
we might want to reuse the same folder and have

https://github.com/confluentinc/ccloud-sdk-go-v2/blob/master/srcm/go.mod#L1

see https://github.com/confluentinc/ccloud-sdk-go-v2/tree/master/flink-gateway setup